orgparse/__init__.py

```diff
@@ -138,25 +139,27 @@ def load(path):
     else:
         orgfile = path
         filename = path.name if hasattr(path, 'name') else '<file-like>'
-    return loadi((l.rstrip('\n') for l in orgfile.readlines()),
-                 filename=filename)
 
+    if not env:
+        env = OrgEnv(filename=filename)
+    return loadi((l.rstrip('\n') for l in orgfile.readlines()), env)
 
-def loads(string, filename='<string>'):
+
+def loads(string, env=OrgEnv(filename='<string>')):
     """
     Load org-mode document from a string.
 
     :rtype: :class:`orgparse.node.OrgRootNode`
 
     """
-    return loadi(string.splitlines(), filename=filename)
+    return loadi(string.splitlines(), env)
 
 
-def loadi(lines, filename='<lines>'):
+def loadi(lines, env=OrgEnv(filename='<lines>')):
```

Comment:
Might be unsafe because OrgEnv object is mutable (and it will be kind of like global variable).
The way you did in [lines 143-144](https://github.com/karlicoss/orgparse/pull/19/files#diff-279135a213a417a10e945ccc26dfeadeR143-R144) is good!

Reply:
True! Will fix.